components/Link.js

```diff
@@ -31,13 +31,7 @@ const Link = ({
   prefetch,
   ...props
 }) => (
-  href.startsWith('http')
-    ? <A {...props} href={href} />
-    : (
-      <NextLink {...props} href={href} prefetch={prefetch}>
-        <A {...props} href={href} />
-      </NextLink>
-    )
+  <A {...props} href={href} />
```

Comment:
@lukebrooker what are the drawbacks of this? I couldn't find any docs on NextLink so I dont know.